src/cbmc/bmc_cover.cpp

```diff
@@ -303,12 +392,21 @@ bool bmc_covert::operator()()
              << (sat_stop-sat_start) << "s" << eom;
   }
   
+
+  // should we minimise testsuite?
+  if(!bmc.options.get_bool_option("disable-testsuite-minimisation"))
+  {
+    testst tests_min;
+    minimise_testsuite(tests, tests_min, goal_map.size());
```

Comment:
I suggest to modify in-place to get rid of the second parameter of minimse_testsuite and copying